components/emoji_picker/modules/emoji_selector.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="d-emoji-picker__selector">
+    <!--    Emoji Selector -->
+    <div
+      v-for="emoji in EMOJI_LIST"
```

Comment:
mhmm... it will be definetly sth from the emoji.json (`node_modules/emoji-toolkit/emoji_strategy.json`)
and/or some utility I will probably create in `common/emoji.js`.

It will be important to have attention on how we will manage this array for the perform impact.
Because we have a bunch of tabset and skins that will affect the search behaviour... not big deal, we can handle it :)